### PR TITLE
practiced-css-basic

### PR DESCRIPTION
**Pull request by Rehmatulla-Khan**

### CODE REVIEW

File: tasks-by-users/coder-mezab/01-css-basic/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>smart phones</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+  </head>
+  <body>
+    <div id="wrap">
```
Comment (AlishaPrasad) [inline tasks-by-users/coder-mezab/01-css-basic/index.html:11]: Where is the </div> tag for this div

File: tasks-by-users/coder-mezab/01-css-basic/css/style.css [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  color: white;
+  text-decoration: none;
+  text-transform: uppercase;
+  border: 1px solid white;
+
+  padding: 6px;
+  font-size: 14px;
+  border-radius: 5px;
+  background: #97781c;
+  background: -webkit-linear-gradient(top, #996800 0%, #ffd34f 100%);
```
Comment (AlishaPrasad) [inline tasks-by-users/coder-mezab/01-css-basic/css/style.css:36]: background is mentioned twice?

File: tasks-by-users/coder-mezab/01-css-basic/css/style.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+#product a:active {
+  background: -webkit-linear-gradient(bottom, #ff35ff 0%, #a500ff 100%);
+}
+.rating {
+  height: 30px;
+  display: block;
+  margin-top: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 25px;
```
Comment (AlishaPrasad) [inline tasks-by-users/coder-mezab/01-css-basic/css/style.css:94]:
```suggestion
  margin: 10px auto 25px;
```

File: tasks-by-users/coder-mezab/01-css-basic/css/style.css [hunk 4]
```diff
@@ -0,0 +1,99 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0%;
+  background-color: #eee;
```
Comment (faran4engg) [inline tasks-by-users/coder-mezab/01-css-basic/css/style.css:4]: good old learning days @coder-mezab :D

File: tasks-by-users/coder-mezab/01-css-basic/index.html (hunk 1 above)
Comment (Naufil-Shemle) [reply tasks-by-users/coder-mezab/01-css-basic/index.html:11]: Hmm👀

**Review by faran4engg — APPROVED** [review]

File: tasks-by-users/coder-mezab/01-css-basic/index.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+      <h2 class="name"> infinix model 2021 </h2>
+      <span class="price">599.99$</span>
+      <a class="add" href="#">Add to card</a>
+      <img class="rating" src="images/rating.png" alt="star rating">
+</section>
+<div  id="foot">
+    <img src="images/foot.jpg" alt="pic of smartphone">
+
+
+</div>
```
[the remark this answers is not in the document]
Comment (coder-mezab) [reply tasks-by-users/coder-mezab/01-css-basic/index.html:33]: @AlishaPrasad fixed all issues

**Review by faran4engg — APPROVED** [review]

**Review by anask601 — APPROVED** [review]

**Pull request merged**